### ISSUE TITLE
Not possible to create a fixed size widget inside a table

### ISSUE THREAD

**Issue by therealjohnsummer** [DavBfr/dart_pdf#151]
I'm using the Table widget to successfully render a grid of data. For example , lets say I am rendering this data : 

USA                    DOLLAR          50
UK                       POUND            70
DENMARK       KRONA             61

This all renders properly and I have 3 rows in the table.

The issue I have is when I try and add another column which is of **fixed width**, i.e I add a container that has a BoxDecoration with all borders rendered. If I try and set the **width** of this container to say 100, the **Table** does not honour it. It seems to size the Table either by the largest Title size or the remaining space for the last column. I basically want to be able to have a column containing a Container of a fixed size with a BoxDecoration and colour, is this supported with the Table widget ?

**Comment by DavBfr** [comment]
Can you give me an example code?

**Comment by therealjohnsummer** [comment]
Issue is resolved. The problem was we wanted to have a Table Header title accompany the Table so that if the Table spanned multiple pages, the Header title would also be spanned. The only way we could accomplish this was to create a TableRow and add it to the table with the **repeat** property set to true. However , this meant that further TableRow's that we added especially container widgets with a specific width were not set correct. 

This can easily be shown with a picture : 

**When we used a TableRow as a header**
![image](https://user-images.githubusercontent.com/54981654/65230809-6c7caa80-dac6-11e9-8b78-52671d24075e.png)

**Working properly**
![image](https://user-images.githubusercontent.com/54981654/65230648-20316a80-dac6-11e9-8f8a-048f909ca73e.png)
